docker/unittest-dedb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM discoenv/facepalm:dev
+
+COPY setup-pg-hbaconf.sh /docker-entrypoint-initdb.d/
+COPY setup-dev-database.sh /bin/setup-dev-database.sh
+COPY update-dev-database.sh /bin/update-dev-database.sh
+COPY pgpass /root/.pgpass
+RUN chmod 0600 /root/.pgpass
+RUN mkdir /data
+ENV POSTGRES_PASSWORD=notprod POSTGRES_USER=de PGDATA=/data
+ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
Where does `docker-entrypoint.sh` come from? I don't see it in these commits, and I can't find it in facepalm's sources...